src/get-billomat-api-client.ts

```diff
@@ -26,6 +26,7 @@ export interface MappedBillomatResourceType {
     articles: Billomat.Article;
     'client-property-values': Billomat.ClientPropertyValue;
     clients: Billomat.Client;
+    confirmations: Billomat.Confirmation;
```

Comment:
I'd like for us to add test data before releasing this. If you want @fxmanu @andrenarchy you can generate and add this, otherwise I'll do so, likely in January.